src/other/Slider.js

```diff
@@ -58,6 +67,40 @@
         return this;
     };
 
+    Slider.prototype.play = function () {
+        var context = this;
+        var tick = this.low(); 
+
+        var intervalHandler = setInterval(function () { 
```

Comment:
If you want to access the intervalHandler from other methods, you will need to make it a property of the Slider Class - so simply change it to:  ```this.intervalHandler```